packages/lesswrong/components/widgets/Snackbar.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import { defineStyles, useStyles } from '../hooks/useStyles';
+import Slide from '@/lib/vendor/@material-ui/core/src/Slide';
+
+export const styles = defineStyles("MuiSnackbar", theme => ({
```

Comment:
this should be `Snackbar` (unless the name here doesn't have to actually match the component name, which is a bit confusing)

Reply:
The names match component names by convention, but this is only convention. In the case of material-UI components, the convention was that they have a `Mui` prefix, and there are preexisting references (eg styles elsewhere referring to `& .MuiSomething`) that expect them to be prefixed with `Mui`.